frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/create-sc/wizard-pages/storage-and-nodes-step.tsx

```diff
@@ -202,6 +205,16 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
           {!!nodesCount && (
             <SelectNodesDetails cpu={cpu} memory={memory} zones={zones.size} nodes={nodesCount} />
           )}
+          <Checkbox
+            label={t('ceph-storage-plugin~Enable taint nodes')}
+            description={t('ceph-storage-plugin~Selected nodes will be dedicated to OCS use only')}
+            id="taint-attached"
+            className="ocs-install_attached__enable-taint"
+            isChecked={state.taintEnabled}
+            onChange={() => {
+              dispatch({ type: 'setTaintEnabled', value: !taintEnabled });
+            }}
+          />
```

Comment:
This component is same therefore I would suggest to create a generic component in `capacity-and-nodes.tsx`
Also create a corresponding scss file `_capacity-and-nodes.scss`

Reply:
Done!